### PR TITLE
Fix the no-op pipeline (for real?)

### PR DESCRIPTION
**Pull request by pierotibou**
The no-op pipeline was running all the time because I didn't disable push triggers (cf [this](https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=azure-devops&tabs=example%2Cparameter-schema#push-trigger)). 

The first and 3rd commit haven't triggered the no-op pipeline, the 2nd one - a line in LICENSE - did trigger it so I think it's working now.

On the 2nd commit, the consolidated pipeline has been triggered as well, but that is because the PR validation pipeline builds the merge commit created by github for the PR, which contains both the LICENSE and .azure-pipelines file. 

I'll do a final test once this PR is merged with a change to a readme file only to validate only the no-op pipeline is triggered.

@DataDog/apm-dotnet

### CODE REVIEW

File: .azure-pipelines/ultimate-pipeline.yml [hunk 1]
```diff
@@ -10,9 +10,14 @@ trigger:
   paths:
     exclude:
       - docs/*
```
Comment (andrewlock) [inline .azure-pipelines/ultimate-pipeline.yml:12]: Wildcards aren't supported!
```suggestion
      - docs/
```

File: .azure-pipelines/ultimate-pipeline.yml [hunk 2]
```diff
@@ -10,9 +10,14 @@ trigger:
...
-      - tracer/README.md
+      - tracer/README.MD
       - .github/*
       - tracer/dependabot/*
+      - tracer/src/Datadog.Trace/DuckTyping/README.md
+      - blog/*
```
Comment (andrewlock) [inline .azure-pipelines/ultimate-pipeline.yml:17]: Wildcards!
```suggestion
      - .github/
      - tracer/dependabot/
      - tracer/src/Datadog.Trace/DuckTyping/README.md
      - blog/
```

**Review by andrewlock — APPROVED** [review]
Third time lucky? 😄 🤞

File: .azure-pipelines/ultimate-pipeline.yml (hunk 2 above)
Comment (pierotibou) [reply .azure-pipelines/ultimate-pipeline.yml:17]: so looking at the doc wildcards are supported in path but only for the push trigger: 
https://docs.microsoft.com/en-us/azure/devops/pipelines/repos/github?view=azure-devops&tabs=yaml#wildcards
that's why I left them

**Comment by andrewlock** [comment]
## Benchmarks Report :snail:

Benchmarks for #2303 compared to [master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550): 
* 2 benchmarks are **faster**, with geometric mean 1.181
* All benchmarks have **the same** allocations


The following thresholds were used for comparing the benchmark speeds:
* Mann–Whitney U test with statistical test for significance of **5%**
* Only results indicating a difference greater than **10%** and **0.3 ns** are considered.

Allocation changes below **0.5%** are ignored.


### Benchmark details


<details><summary><strong>Benchmarks.Trace.AgentWriterBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`WriteAndFlushEnrichedTraces`|net472|908μs|2.97μs|11.1μs|0| 0|0|3.16&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`WriteAndFlushEnrichedTraces`|netcoreapp3.1|706μs|1.66μs|5.97μs|0| 0|0|2.57&nbsp;KB|
|#2303|`WriteAndFlushEnrichedTraces`|net472|869μs|2.48μs|9.6μs|0| 0|0|3.16&nbsp;KB|
|#2303|`WriteAndFlushEnrichedTraces`|netcoreapp3.1|675μs|2.97μs|13.6μs|0| 0|0|2.57&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.AspNetCoreBenchmark</strong> - Faster :tada: Same allocations :heavy_check_mark:



#### Faster :tada: in #2303

|  Benchmark         | base/diff | Base Median (ns) | Diff Median (ns) | Modality   |
|:----------|-----------:|-----------:|--------:|--------:|
| Benchmarks.Trace.AspNetCoreBenchmark.SendRequest&#8209;netcoreapp3.1 | 1.174 | 344,666.27 | 293,466.68 | 

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`SendRequest`|net472|0ns|0ns|0ns|0| 0|0|0&nbsp;b|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`SendRequest`|netcoreapp3.1|347μs|1.63μs|6.31μs|0.171| 0|0|19.63&nbsp;KB|
|#2303|`SendRequest`|net472|0ns|0ns|0ns|0| 0|0|0&nbsp;b|
|#2303|`SendRequest`|netcoreapp3.1|294μs|941ns|3.39μs|0.151| 0|0|19.63&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.DbCommandBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`ExecuteNonQuery`|net472|1.72μs|8.58ns|61.2ns|0.0923| 0.000832|0|586&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`ExecuteNonQuery`|netcoreapp3.1|1.48μs|6.24ns|24.2ns|0.00894| 0|0|624&nbsp;B|
|#2303|`ExecuteNonQuery`|net472|1.62μs|5.67ns|22.7ns|0.093| 0.000802|0|586&nbsp;B|
|#2303|`ExecuteNonQuery`|netcoreapp3.1|1.61μs|4.98ns|18.6ns|0.00869| 0|0|624&nbsp;B|
</details>


<details><summary><strong>Benchmarks.Trace.ElasticsearchBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`CallElasticsearch`|net472|2.6μs|13.4ns|64.1ns|0.123| 0|0|794&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`CallElasticsearch`|netcoreapp3.1|1.8μs|10.4ns|101ns|0.0109| 0|0|784&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`CallElasticsearchAsync`|net472|3.04μs|16.3ns|86.4ns|0.145| 0|0|931&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`CallElasticsearchAsync`|netcoreapp3.1|1.82μs|6.1ns|22ns|0.0128| 0|0|904&nbsp;B|
|#2303|`CallElasticsearch`|net472|2.54μs|12ns|49.4ns|0.123| 0|0|794&nbsp;B|
|#2303|`CallElasticsearch`|netcoreapp3.1|1.65μs|5.97ns|23.1ns|0.0104| 0|0|784&nbsp;B|
|#2303|`CallElasticsearchAsync`|net472|2.72μs|15.4ns|113ns|0.145| 0|0|931&nbsp;B|
|#2303|`CallElasticsearchAsync`|netcoreapp3.1|1.79μs|7.91ns|30.6ns|0.0122| 0|0|904&nbsp;B|
</details>


<details><summary><strong>Benchmarks.Trace.GraphQLBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`ExecuteAsync`|net472|2.89μs|11.6ns|44.9ns|0.164| 0|0|1.05&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`ExecuteAsync`|netcoreapp3.1|2.06μs|8.43ns|32.6ns|0.0141| 0|0|1.02&nbsp;KB|
|#2303|`ExecuteAsync`|net472|2.8μs|13.7ns|54.7ns|0.164| 0|0|1.05&nbsp;KB|
|#2303|`ExecuteAsync`|netcoreapp3.1|1.95μs|10.2ns|50ns|0.0141| 0|0|1.02&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.HttpClientBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`SendAsync`|net472|6.47μs|27.4ns|102ns|0.352| 0|0|2.26&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`SendAsync`|netcoreapp3.1|4.68μs|18.9ns|77.7ns|0.0299| 0|0|2.16&nbsp;KB|
|#2303|`SendAsync`|net472|6.48μs|29.8ns|112ns|0.353| 0|0|2.26&nbsp;KB|
|#2303|`SendAsync`|netcoreapp3.1|4.4μs|14.8ns|57.4ns|0.0287| 0|0|2.16&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.ILoggerBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`EnrichedLog`|net472|3.54μs|19.5ns|126ns|0.225| 0|0|1.44&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`EnrichedLog`|netcoreapp3.1|3.26μs|13.5ns|52.4ns|0.021| 0|0|1.52&nbsp;KB|
|#2303|`EnrichedLog`|net472|3.36μs|17.4ns|86.8ns|0.224| 0|0|1.44&nbsp;KB|
|#2303|`EnrichedLog`|netcoreapp3.1|3.15μs|15.5ns|66ns|0.0211| 0|0|1.52&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.Log4netBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`EnrichedLog`|net472|216μs|1.26μs|11.6μs|0.428| 0.107|0|4.31&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`EnrichedLog`|netcoreapp3.1|189μs|1.1μs|9.48μs|0| 0|0|4.19&nbsp;KB|
|#2303|`EnrichedLog`|net472|199μs|942ns|4μs|0.497| 0.199|0|4.31&nbsp;KB|
|#2303|`EnrichedLog`|netcoreapp3.1|170μs|919ns|5.12μs|0| 0|0|4.19&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.NLogBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`EnrichedLog`|net472|7.85μs|28.7ns|107ns|0.502| 0|0|3.21&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`EnrichedLog`|netcoreapp3.1|6.69μs|23.4ns|87.7ns|0.0496| 0|0|3.58&nbsp;KB|
|#2303|`EnrichedLog`|net472|7.73μs|36.7ns|151ns|0.501| 0|0|3.21&nbsp;KB|
|#2303|`EnrichedLog`|netcoreapp3.1|6.64μs|22.6ns|118ns|0.0479| 0|0|3.58&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.RedisBenchmark</strong> - Faster :tada: Same allocations :heavy_check_mark:



#### Faster :tada: in #2303

|  Benchmark         | base/diff | Base Median (ns) | Diff Median (ns) | Modality   |
|:----------|-----------:|-----------:|--------:|--------:|
| Benchmarks.Trace.RedisBenchmark.SendReceive&#8209;net472 | 1.187 | 2,562.12 | 2,158.57 | 

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`SendReceive`|net472|2.58μs|13.1ns|60ns|0.157| 0|0|1&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`SendReceive`|netcoreapp3.1|2.15μs|9.1ns|34.1ns|0.0137| 0|0|1&nbsp;KB|
|#2303|`SendReceive`|net472|2.17μs|7.62ns|29.5ns|0.157| 0|0|1&nbsp;KB|
|#2303|`SendReceive`|netcoreapp3.1|1.97μs|9.1ns|37.5ns|0.0137| 0|0|1&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.SerilogBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`EnrichedLog`|net472|7.19μs|37.8ns|272ns|0.287| 0|0|1.85&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`EnrichedLog`|netcoreapp3.1|5.46μs|20ns|71.9ns|0.0188| 0|0|1.47&nbsp;KB|
|#2303|`EnrichedLog`|net472|6.96μs|37.2ns|211ns|0.287| 0|0|1.85&nbsp;KB|
|#2303|`EnrichedLog`|netcoreapp3.1|5.41μs|31.3ns|254ns|0.0188| 0|0|1.47&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.SpanBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`StartFinishSpan`|net472|997ns|5.73ns|42.8ns|0.0705| 0|0|449&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`StartFinishSpan`|netcoreapp3.1|1μs|4.62ns|17.9ns|0.0064| 0|0|448&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`StartFinishScope`|net472|1.18μs|6.24ns|31.2ns|0.083| 0|0|530&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`StartFinishScope`|netcoreapp3.1|1.11μs|3.64ns|14.5ns|0.00772| 0|0|568&nbsp;B|
|#2303|`StartFinishSpan`|net472|960ns|4.88ns|23.4ns|0.0706| 0|0|449&nbsp;B|
|#2303|`StartFinishSpan`|netcoreapp3.1|941ns|3.8ns|14.7ns|0.00605| 0|0|448&nbsp;B|
|#2303|`StartFinishScope`|net472|1.14μs|5.51ns|21.3ns|0.0832| 0|0|530&nbsp;B|
|#2303|`StartFinishScope`|netcoreapp3.1|1.08μs|4.48ns|17.3ns|0.00753| 0|0|568&nbsp;B|
</details>

File: .azure-pipelines/ultimate-pipeline.yml (hunk 2 above)
Comment (pierotibou) [reply .azure-pipelines/ultimate-pipeline.yml:17]: Yes I agree. As those were already there originally, I didn't want to do any more changes than necessary TBH. 2 failed attempts are already too much

Comment (andrewlock) [reply .azure-pipelines/ultimate-pipeline.yml:17]: > so looking at the doc wildcards are supported in path but only for the push trigger

What were they thinking 🤦‍♂️

**Comment by andrewlock** [comment]
## Code Coverage Report :bar_chart:

:heavy_check_mark: Merging #2303 into [master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550) will **not change** line coverage
:heavy_check_mark: Merging #2303 into [master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550) will **not change** branch coverage
:heavy_check_mark: Merging #2303 into [master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550) will **not change** complexity


|           | [master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550) | #2303       | Change   | 
|:----------|:-----------:|:-----------:|:--------:|
| Lines     | `10597` / `14831` | `10558` / `14831` |          |
| Lines %   | `71%`      | `71%`      |  `0%` :heavy_check_mark:  |
| Branches  | `5322` / `7675` | `5298` / `7675` |          |
| Branches %| `69%`      | `69%`      |  `0%` :heavy_check_mark:  |
| Complexity|   `8914`      | `8914`        |  `0`  :heavy_check_mark:    |

View the full report for further details:

* [HTML report for master](https://dev.azure.com/datadoghq/dd-trace-dotnet/_build/results?buildId=86992&view=codecoverage-tab) | [Source file](https://dev.azure.com/datadoghq/a51c4863-3eb4-4c5d-878a-58b41a049e4e/_apis/build/builds/86992/artifacts?artifactName=Code%20Coverage%20Report_86992&api-version=6.0&%24format=zip)  
* [HTML report for this PR #2303](https://dev.azure.com/datadoghq/dd-trace-dotnet/_build/results?buildId=87022&view=codecoverage-tab) | [Source file](https://dev.azure.com/datadoghq/a51c4863-3eb4-4c5d-878a-58b41a049e4e/_apis/build/builds/87022/artifacts?artifactName=Code%20Coverage%20Report_87022&api-version=6.0&%24format=zip)

### Datadog.Trace Breakdown :heavy_check_mark:

|        | [master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550) | #2303       | Change   | 
|:-------|:-----------:|:-----------:|:--------:|
| Lines %| `71%`      | `71%`       |  `0%` :heavy_check_mark:  |
| Branches %| `69%`      | `69%`       |  `0%` :heavy_check_mark:  |
| Complexity| `8914`      | `8914`       |  `0` :heavy_check_mark:  |

The following classes have significant coverage changes.

| File    | Line coverage change | Branch coverage change | Complexity change |
|:--------|:--------------------:|:----------:|:--------:|
| [Datadog.Trace.ClrProfiler.AutoInstrumentation.Couchbase.IIOServiceExecuteIntegrationBis](https://github.com/DataDog/dd-trace-dotnet/tree/fcff86db2f084f87a92e07679c0b6ddd29328193/tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Couchbase/IIOServiceExecuteIntegrationBis.cs) | `-100%` :no_entry: | `0%` :heavy_check_mark:   | `0` :heavy_check_mark: |
| [Datadog.Trace.ClrProfiler.AutoInstrumentation.Testing.MsTestV2.TestMethodRunnerExecuteIntegration](https://github.com/DataDog/dd-trace-dotnet/tree/fcff86db2f084f87a92e07679c0b6ddd29328193/tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Testing/MsTestV2/TestMethodRunnerExecuteIntegration.cs) | `-33%` :no_entry: | `-19%` :no_entry:   | `0` :heavy_check_mark: |
| [Datadog.Trace.Ci.Agent.CIAgentWriter](https://github.com/DataDog/dd-trace-dotnet/tree/fcff86db2f084f87a92e07679c0b6ddd29328193/tracer/src/Datadog.Trace/Ci/Agent/CIAgentWriter.cs) | `-10%` :no_entry: | `-21%` :no_entry:   | `0` :heavy_check_mark: |
| [Datadog.Trace.ClrProfiler.AutoInstrumentation.Testing.NUnit.NUnitCompositeWorkItemSkipChildrenIntegration](https://github.com/DataDog/dd-trace-dotnet/tree/fcff86db2f084f87a92e07679c0b6ddd29328193/tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Testing/NUnit/NUnitCompositeWorkItemSkipChildrenIntegration.cs) | `-8%` :no_entry: | `-8%` :no_entry:   | `0` :heavy_check_mark: |


View the full reports for further details:

* [HTML report for master](https://dev.azure.com/datadoghq/dd-trace-dotnet/_build/results?buildId=86992&view=codecoverage-tab) | [Source file](https://dev.azure.com/datadoghq/a51c4863-3eb4-4c5d-878a-58b41a049e4e/_apis/build/builds/86992/artifacts?artifactName=Code%20Coverage%20Report_86992&api-version=6.0&%24format=zip)  
* [HTML report for this PR #2303](https://dev.azure.com/datadoghq/dd-trace-dotnet/_build/results?buildId=87022&view=codecoverage-tab) | [Source file](https://dev.azure.com/datadoghq/a51c4863-3eb4-4c5d-878a-58b41a049e4e/_apis/build/builds/87022/artifacts?artifactName=Code%20Coverage%20Report_87022&api-version=6.0&%24format=zip)

**Pull request merged**